### PR TITLE
Issue 16

### LINKED ISSUE #9: Increase size of fields so it can show all content
For the actual payload values maybe use the max LoRaWAN payload size as the length. Downlinks will not exceed a payload length of 52 bytes.

Example fields that are to small:
- GRPC Host:Port
- several fields in the settings tab
- several fields in the commands tab

### LINKED ISSUE #16: Allow users to select multiple devices from the Select Device section.
The applications then should trigger separate messages to each of the selected devices. We do not want to use the special function of Chirpstack to send to multiple devices at once. We just want to fire multiple messages.

### PR DESCRIPTION
**Pull request by bsuligoj**
Added support for multiple device select. 
Code was reformatted a bit.
Removed unused variables.
Resized input fields.

closes #16 
closes #9

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
when there is an error please also display the description in the popup box like for example 
SP050005: Error - object does not exist

Also if not too much hassle put the popup box in the middle of the page

In the Request Records tab the payload hex is missing

for the input boxes please move them in this order to make it more organized in terms of space
The last field should be the input and should be fluid to fill the rest of the remaining page width

photo_2023-04-05_13-36-20

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+.env*
```
Comment (krasi-georgiev) [inline .gitignore:2]: What do we keep in the .env file?

File: public/src/assets/menu.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.hamburger-menu-container {
```
Comment (krasi-georgiev) [inline public/src/assets/menu.css:1]: There was some padding arround the hamb menu and it is lost. 

Please add like 10px padding from the top and left of the page

File: .gitignore (hunk 1 above)
Comment (bsuligoj) [reply .gitignore:2]: In case anyone creates .env for development purpose etc, it's a precaution to not accidentally put it in git

**Comment by bsuligoj** [comment]
> when there is an error please also display the description in the popup box like for example SP050005: Error - object does not exist
> 
> Also if not too much hassle put the popup box in the middle of the page
> 
> In the Request Records tab the payload hex is missing
> 
> for the input boxes please move them in this order to make it more organized in terms of space The last field should be the input and should be fluid to fill the rest of the remaining page width
> 
> photo_2023-04-05_13-36-20

Regarding the error, I will return response that is returned by the backend, it is impractical and counter pattern to change error messages on the frontend side.

Regarding the message, in this format, it cannot be moved in the middle, but it can be done with e.g. https://element-plus.org/en-US/component/message-box.html

Regarding payload in Hex, functionality was not changed and it is not working from before since backend is not returning it. 
Edit: It is not working due typo **Paylod**, missing **a**. https://github.com/SmartParksOrg/smartparks-connect-web/blob/main/public/src/components/Form.vue#L195

I will reorder the fields in requests and command tabs.

**Comment by krasi-georgiev** [comment]
> Regarding the error, I will return response that is returned by the backend, it is impractical and counter pattern to change error messages on the frontend side.

yes this is what I mean, last time I tried it didn't return the error from the backend

> Regarding the message, in this format, it cannot be moved in the middle, but it can be done with e.g. [element-plus.org/en-US/component/message-box.html](https://element-plus.org/en-US/component/message-box.html)

if not too much hassle then let's use that one.

**Pull request merged**